package.json

```
@@ -18,7 +18,7 @@
     "fix-lint": "eslint --fix --ext .ts,.js,.vue . && npm run lint:style -- --fix"
   },
   "engines": {
-    "node": ">=14.x"
+    "node": "=14.x"
```

```suggestion
    "node": "14.x"
```

Reading the docs from [Heroku: Specifying a Node.js Version](https://devcenter.heroku.com/articles/nodejs-support#specifying-a-node-js-version) I think it should be without the `=`, right?.